cross/toolchain.cmake

```diff
@@ -0,0 +1,154 @@
+set(CROSS_ROOTFS $ENV{ROOTFS_DIR})
+
```

Comment:
Can you move "cross" to eng/cross. That's where we try to keep all the build/engineering stuff.

Reply:
Sure